src/Hipace.cpp

```diff
@@ -673,26 +670,23 @@ Hipace::ExplicitSolveBxBy (const int lev)
 
     // preparing conversion to normalized units, if applicable
     PhysConst pc = m_phys_const;
-    const amrex::Real n0 = m_multi_plasma.GetNominalDensity();
```

Comment:
For the cleaning: could you ensure that `GetNominalDensity` and the corresponding `m_nominal_density` is also removed?